src/Configuration.ts

```diff
@@ -22,4 +23,9 @@ export class Configuration {
         const cfg = vscode.workspace.getConfiguration(TTDBG_CONFIG_SECTION, folder);
         return cfg.get<number>(DEBUG_PROXY_PORT, 2345);
     }
+
+    static getJustMyCode(folder?: vscode.WorkspaceFolder) {
```

Comment:
Is this setting to prevent the debugger going into the DBOS library code? As long as we document how to change it to `false` we're good.

Reply:
Yup, maps to official python `justMyCode` [debug config option](https://code.visualstudio.com/docs/python/debugging#_justmycode). Most users will want the default (true) which will preclude them from stepping thru DBOS library code. However, some folks (esp us on the DBOS team) will want to be able to step thru the DBOS code when debugging as well.